cl/users/templates/includes/webhook-event-detail.html

```diff
@@ -0,0 +1,73 @@
+                  <label class="col-md-2">Event ID: </label><p class="col-md-10">{{ webhook_event.event_id }}</p>
+                  <label class="col-md-2">Date created: </label><p class="col-md-10">{{ webhook_event.date_created }}</p>
+                  <label class="col-md-2">Status code: </label><p class="col-md-10 bold">{{ webhook_event.status_code }}</p>
+                  <label class="col-md-2">Event status: </label><p class="col-md-10 bold status-{{ webhook_event.event_status }}">{{ webhook_event.get_event_status_display }}</p>
+                  <label class="col-md-2">Try counter: </label><p class="col-md-10">{{ webhook_event.retry_counter }}</p>
+                  <label class="col-md-2">Next retry date: </label><p class="col-md-10">{% if not webhook_event.debug %}{{ webhook_event.next_retry_date }}{% else %}Test events will not be retried{% endif %}</p>
+                </div>
+                <div class="row form-group">
+                  <div class="col-xs-12">
+                    <label for="content">Content:</label>
+                    <div class="dummy-area">
+                      <div class="copy-text-icon" onclick="copy_text('content')"><i class="fa fa-copy"></i></div>
+                      <textarea readonly class="form-control" id="content" rows="8">{{ json_content }}</textarea>
+                    </div>
+                  </div>
+                </div>
+                <div class="row form-group">
+                  <div class="col-xs-12">
+                    <label for="response">Response:</label>
+                    <div class="dummy-area">
+                      <div class="copy-text-icon" onclick="copy_text('response')"><i class="fa fa-copy"></i></div>
+                      <textarea readonly class="form-control" id="response" rows="8">{{ webhook_event.response }}</textarea>
+                    </div>
+                  </div>
+                </div>
+                <div class="row form-group">
+                    <div class="col-xs-12">
+                      <label for="error_message">Error message:</label>
+                      <div class="dummy-area">
+                        <div class="copy-text-icon" onclick="copy_text('error_message')"><i class="fa fa-copy"></i></div>
+                        <textarea readonly class="form-control" id="error_message" rows="4">{{ webhook_event.error_message }}</textarea>
+                      </div>
+                    </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      {% include "includes/webhooks_htmx/webhooks-modal.html" %}
+      <p>Webhooks enable bi-directional APIs by sending data to your server whenever events happen on ours. Use webhooks to get the fastest updates &mdash; without polling.</p>
+      <p class="v-offset-below-2"><a href="{% url "api_index" %}#webhooks" class="btn btn-default btn-lg">Read the Webhook Documentation</a></p>
+    </div>
```

Comment:
Surprised to see this here. Should be abstracted out.

Reply:
Yes, I moved all this common code to webhooks_base.html so it's not repeated.